doorstop/cli/main.py

```diff
@@ -221,6 +231,18 @@ def _add(subs, shared):
     )
     sub.add_argument('prefix', help="document prefix for the new item")
     sub.add_argument('-l', '--level', help="desired item level (e.g. 1.2.3)")
+    sub.add_argument(
+        '-n',
+        '--name',
+        '--number',
```

Comment:
Could you add the new `--separator`, `--name`, and `--number` options as an examples in [`doorstop/docs/cli/creation.md`](https://github.com/doorstop-dev/doorstop/blob/174d1399ea46864188b37e27fd14331f8ed1c770/docs/cli/creation.md)?